app/api/auth/logout.feature

```diff
@@ -6,16 +6,55 @@ Feature: Sign the current user out
       | 3        | jane  |
     And the DB time now is "2019-07-16 22:02:28"
     And the database has the following table 'sessions':
+      | user_id | expires_at          | access_token              | use_cookie |
```

Comment:
Should we handle the case when a token is given through headers and cookies? 
In particular I can think of a version of the app which uses cookies, then the app use header token... but when login out, if the cookie is not removed, it will be served for next requests (instead of a 401), which will cause strange behaviors quite hard to debug.

Reply:
What will happen to the header in that case? If the header remains, the cookie is not used (even when the header contains a wrong token)

Reply:
When there is no token, no authorization header is sent by the frontend. In practice, it always tries to get a new one but nothing really guarantees that. Maybe that's not so harmful, indeed (especially if we use quite short validity for tokens)...
Do you have scenarios testing that? That we set a cookie, then use a valid token in headers (must use the token), then an invalid token in headers (must use the token).

Reply:
As I wrote in the comment below, actually that only work if the create-temp-user and create-token-from-code work even if a cookie is provided (and ignored) tough.